debian/control

```diff
@@ -0,0 +1,20 @@
+Source: golang-logrus
+Section: devel
+Priority: extra
+Maintainer: Jessie Frazelle <jess@docker.com>
+Uploaders: Docker Packaging Team <docker-maint@lists.alioth.debian.org>
+Build-Depends: debhelper (>= 9), dh-golang, golang-go, golang-go.net-dev
+Standards-Version: 3.9.5
+Homepage: https://github.com/Sirupsen/logrus
+Vcs-Git: git://anonscm.debian.org/pkg-go/packages/golang-logrus.git
+Vcs-Browser: http://anonscm.debian.org/cgit/pkg-go/packages/golang-logrus.git
+
+Package: golang-logrus-dev
+Architecture: all
+Depends: golang-go.net-dev, ${misc:Depends}, ${shlibs:Depends}
+Built-Using: ${misc:Built-Using}
```

Comment:
because this is Arch: `all`, we don't have any binaries in here -- we don't need that `built-using`, AFAIK -- any reason that's there or was it just copy-paste :)